mitodl/open-discussions
# Add notification settings model and API

#### What are the relevant tickets?
Fixes #459 

#### What's this PR do?
Adds settings API for email nottifications

#### How should this be manually tested?

- Go to `/api/v0/notification_settings/`, verify you get an empty list
- Run `./manage.py populate_notification_settings --username USERNAME`, verify you get a `NotificationSetting` record for the frontpage
- Go to `/api/v0/notification_settings/`, verify you get that same setting in the list
- Go to `/api/v0/notification_settings/frontpage/`, verify you get that same setting
- Verify you can patch the setting with a new `trigger_frequency` value
